Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1048,6 +1047,19 @@ final class StatusTableViewController: ChartsTableViewController {
         }
     }
 
+    private func presentErrorCancelingBolus(_ error: (Error)) {
+        self.log.error("Error Canceling Bolus: %@", error.localizedDescription)
+        let title = NSLocalizedString("Error Canceling Bolus", comment: "The alert title for an error while canceling a bolus")
+        let body = NSLocalizedString("""
+                                    The app was unable to stop the bolus in progress. Move your iPhone closer to the pump and try again. Confirm total insulin delivered in your insulin delivery history and monitor your glucose closely.
```

Comment:
Ideally we want bolus cancellation to report a localizable error, with a reason and recovery suggestion, in which case, we wouldn't put this message here. We can consider that out of scope for this ticket, for now, though.

Reply:
Also, multiline literal for a single line of text seems a little weird, but that's a tiny nit.

Reply:
@ps2 totally agree that this could be better.  This is localizable, but as you point out, the recovery suggestion is not based on the reason for the failure, but that wasn't part of this ticket.
I'll fix the "multiline" text to be a single line (I don't know why I did it this way...)